include/util/Base64.h

```diff
@@ -0,0 +1,67 @@
+/**
+ *
+ * @copyright Copyright (c) 2022 ApertureData Inc.
+ *
+ */
+
+#pragma once
+
+#include <string>
+#include <vector>
+#include <cassert>
+#include <openssl/evp.h>
+
+class Base64 {
+public:
+    // static methods only
+    Base64() = delete;
+
+    static constexpr std::size_t encoded_bytes( std::size_t bytes )
```

Comment:
Can you write a comment that these are not inverses in the code, because it seems strange.